std/conv.d

```diff
@@ -3340,6 +3340,10 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source) && !is(Source == enum
     }
     r = to!real(to!string(real.max));
     assert(to!string(r) == to!string(real.max));
+
+    real pi = 3.1415926535897932384626433832795028841971693993751;
+    string full_precision = "3.1415926535897932384626433832795028841971693993751";
+    assert(feq(parse!real(full_precision), pi, 2*real.epsilon));
```

Comment:
This finally exercises the extended precision code with `lsdec` above, which [codecov.io says wasn't covered](https://codecov.io/gh/dlang/phobos/src/master/std/conv.d#L3147).